hudi-client/hudi-client-common/src/test/java/org/apache/hudi/table/action/TestCleanPlanner.java

```diff
@@ -160,6 +178,12 @@ void testPartitionsForIncrCleaning(HoodieWriteConfig config, String earliestInst
     mockLastCleanCommit(mockHoodieTable, lastCleanInstant, earliestInstantsInLastClean, activeTimeline, cleanMetadataOptionPair);
     mockFewActiveInstants(mockHoodieTable, activeInstantsPartitions, savepointsTrackedInLastClean, areCommitsForSavepointsRemoved);
 
+    // mock getAllPartitions
+    HoodieStorage storage = mock(HoodieStorage.class);
+    when(mockHoodieTable.getStorage()).thenReturn(storage);
+    mockedStatic.when(() -> FSUtils.getAllPartitionPaths(context, storage, config.getMetadataConfig(), config.getBasePath()))
```

Comment:
we could also update the CleanPlanner to use `hoodieTable.getMetadataTable().getAllPartitionPaths()` which could make the test setup cleaner as well